qutebrowser/browser/pdfjs.py

```diff
@@ -49,7 +49,8 @@ def generate_pdfjs_page(filename, url):
         filename: The filename of the PDF to open.
         url: The URL being opened.
     """
-    if not is_available():
+    pdfjs_bin = is_available()
```

Comment:
```suggestion
    pdfjs_name = is_available()
```

or somesuch - it's not a binary after all, so that name seems a bit confusing.

Reply:
The variable has been renamed and is more expressive now.